src/Umbraco.Web.UI.Client/src/views/components/forms/umb-checkbox.html

```diff
@@ -15,5 +15,12 @@
             <i class="umb-form-check__icon icon-check"></i>
         </span>
     </span>
-    <span class="umb-form-check__text">{{vm.text}}</span>
+
+    <!-- If there is no localize attribute set then render the vm.text -->
+    <span class="umb-form-check__text" ng-if="vm.labelKey === undefined">{{vm.text}}</span>
+
+    <!-- If there is a localize attribute set then render the vm.localize with a fallback to vm.text -->
+    <span class="umb-form-check__text" ng-if="vm.labelKey.length > 0">
+        <localize key="{{::vm.labelKey}}">{{vm.text}}</localize>
+    </span>
```

Comment:
@BatJan I think it would be better to handle this in the controller to localize `vm.text` and keep this in the view: `<span class="umb-form-check__text">{{vm.text}}</span>`.

Then use `localizationService` to localize text here:
https://github.com/umbraco/Umbraco-CMS/blob/v8/dev/src/Umbraco.Web.UI.Client/src/common/directives/components/forms/umbcheckbox.directive.js#L39-L52

For example:

```
function init() {
    if (vm.labelKey) {
         localizationService.localize(key).then(function (data) {
              vm.text = data;
         });
    }
}

init();
```

Reply:
@bjarnef Yes, that's a good point, But I would like to note that for another PR as I really want to have this merged so I can do smaller ones afterwards creating less headaches and memory loss in case they will be open for a while 😃 (Aging you know)